java/doc/installation.rst

```diff
@@ -46,7 +46,7 @@ Then we can start building Ray with the following commands:
   ci/travis/install-bazel.sh
 
   # build native components
-  ./build.sh -l java
+  ./build.sh -l java  # TODO(pcmoritz)
```

Comment:
We can create a small build.sh here that calls `RAY_INSTALL_JAVA=1 pip install -e .` in the java case and `pip install -e .` in the python case here to support some people's workflow :)